lesson07/script.js

```diff
@@ -27,8 +27,8 @@ function createList(value){
 }
 
 function getList(){
-    return new Promise(resolve => {  
     createLoading();
+    return new Promise(resolve => {  
     const lists = [
         {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
         {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
```

Comment:
i put createLoading function outside Promise.
Because createLoading should be outside of promise.